plugins/uefi-capsule/efi/meson.build

```diff
@@ -154,17 +154,25 @@ so = custom_target('fwup.so',
                              efi_ldflags + ['@INPUT@'] +
                              ['-lefi', '-lgnuefi', libgcc_file_name])
 
+sbat_csv = configure_file(
+  input : 'sbat.csv.in',
+  output : 'sbat.csv',
+  encoding : 'utf-8',
+  configuration : conf
+)
+
 configure_file(
-  input : 'generate_binary.sh.in',
-  output : 'generate_binary.sh',
+  input : 'generate_binary.py',
```

Comment:
this whole `configure_file` section should be dropped no?

Reply:
I think it will still be needed or otherwise the script won't be copied to the build dir ?

The `sbat.csv` config file will be removed though. Sorry that I didn't re-spin this PR yet. I will try to do it later today or tomorrow.

Reply:
Does it need to be copied to build sir though? I would think if it just executed from source dir that should be fine.

Reply:
I see. I think you are correct. I'll look at this too when addressing the other comments.